pkg/cmd/cli/cli.go

```diff
@@ -101,7 +101,7 @@ func NewCommandCLI(name, fullName string, out io.Writer) *cobra.Command {
 				cmd.NewCmdDescribe(fullName, f, out),
 				cmd.NewCmdEdit(fullName, f, out),
 				cmd.NewCmdEnv(fullName, f, in, out),
-				cmd.NewCmdVolume(fullName, f, out),
+				cmd.NewCmdVolume(fullName, f, out, errout),
```

Comment:
+1, Other cmds like NewCmdEnv, etc. should take errout and use errout stream for errors.
do we want to create a separate issue for that?